actions/delegator/setup-token/action.yml

```diff
@@ -42,6 +42,18 @@ inputs:
     type: string
     required: true
 
+  slsa-workflow-inputs-mask:
```

Comment:
Why not name it `slsa-workflow-masked-inputs` to be consistent?

Reply:
works for me.